src/views/base.njk

```diff
@@ -19,6 +19,7 @@
 
 {% block main %}
   <div class="govuk-width-container">
+    {% block beforeContent %}{% endblock %}
```

Comment:
This can be removed, as there isn't any content to be placed within the `beforeContent` block